lib/suspenders/app_builder.rb

```diff
@@ -375,6 +375,21 @@ def copy_dotfiles
       directory("dotfiles", ".")
     end
 
+    def download_style_guides
+      download_file(
+        ".jscsrc",
+        "https://raw.githubusercontent.com/thoughtbot/guides/master/style/javascript/.jscsrc",
```

Comment:
What do you think about extracting something alongside the lines of `config_url("thoughtbot/guides", "style/javascript/.jscsrc")` ?